sheetstorm/management/commands/generate_test_data.py

```diff
@@ -1,15 +1,23 @@
 import logging
 
+from dateutil.relativedelta import relativedelta
 from django.contrib.auth import get_user_model
 from django.core.management.base import BaseCommand
 from django.db import transaction
+from django.utils import timezone
 
+from managers.factories import ProjectFactory
+from managers.models import Project
 from users.factories import UserFactory
 
 CustomUser = get_user_model()
 
 superuser_user_type = "SUPERUSER"
 
+suspended_project = "suspended"
+active_project = "active"
+completed_project = "completed"
```

Comment:
Please create from these variables enum class

Reply:
Do you want me to add this class in this file or create new class in Project class from managers/models.py like this is solved in CustomUser class case?

Reply:
Done